Sources/BraintreePayPal/BTPayPalCheckoutRequest.swift

```diff
@@ -98,21 +83,30 @@ import BraintreeCore
     ///   See https://developer.paypal.com/docs/api/reference/currency-codes/ for a list of supported currency codes.
     ///   - requestBillingAgreement: Optional: If set to `true`, this enables the Checkout with Vault flow, where the customer will be prompted to consent to a billing agreement
     ///   during checkout. Defaults to `false`.
+    ///   - userAuthenticationEmail: Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
+    ///   - userPhoneNumber: Optional: A user's phone number to initiate a quicker authentication flow in the scenario where the user has a PayPal account
+    /// identified with the same phone number.
+    ///   - lineItems: Optional: The line items for this transaction. It can include up to 249 line items.
     public init(
-        amount: String,
-        intent: BTPayPalRequestIntent = .authorize,
+        amount: String?,
+        intent: BTPayPalRequestIntent? = .authorize,
```

Comment:
General question, have we checked the Obj-C interface for this? Just want to verify the optional enum is okay there.